src/SFA.DAS.EmployerIncentives.Commands/ApprenticeshipIncentive/RefreshLearner/RefreshLearnerCommandHandler.cs

```diff
@@ -63,7 +63,7 @@ public async Task Handle(RefreshLearnerCommand command, CancellationToken cancel
                 submissionData.SetRawJson(learnerData.RawJson);
             }
 
-            if (!submissionData.HasChangeOfCircumstances(learner.SubmissionData))
+            if (submissionData.HasChangeOfCircumstances(learner.SubmissionData))
```

Comment:
was this a bug?

Reply:
It was the wrong way round.  Was returning false when the data had a change of circumstance rather than true so was being used the wrong way round too.